### PR TITLE
AGS: Add language detection (N-O) + add few missing games

### PR DESCRIPTION
**Pull request by tag2015**

### CODE REVIEW

File: engines/ags/detection_tables.h [hunk 1]
```diff
@@ -1920,6 +1923,9 @@ const PlainGameDescriptor GAME_NAMES[] = {
 	{ "osdscourge", "Oceanspirit Dennis: Scourge of the Underworld" },
 	{ "osdscourgehd", "Oceanspirit Dennis: Scourge of the Underworld HD" },
 	{ "osdsearch", "The Search for Oceanspirit Dennis" },
+	{ "osdshindig", "The Shindig of Dennis" },
+	{ "osdsquares", "Oceanspirit Dennis: [][][][][][][][][]" },
```
Comment (bluegr) [inline engines/ags/detection_tables.h:1927]: Are these part of the actual game title?

Comment (tag2015) [reply engines/ags/detection_tables.h:1927]: Yes it's weird but that's the name on the official website
http://www.barnrunner.com/OSD/ (see number #15)
The squares are unicode 25AF, but that character doesn't show on scummvm gamelist, so I used the square brackets.

Comment (bluegr) [reply engines/ags/detection_tables.h:1927]: Thanks! Odd naming...

**Comment by bluegr** [comment]
Thanks for your work! Merging

**Pull request merged**